x-pack/plugins/fleet/common/types/rest_spec/epm.ts

```diff
@@ -31,7 +31,7 @@ export interface GetCategoriesResponse {
 export interface GetPackagesRequest {
   query: {
     category?: string;
-    experimental?: boolean;
+    prerelease?: boolean;
```

Comment:
Renaming `experimental` flag to `prerelease` is a breaking change (that is why I had to update other plugins using this API). It might impact other repos as well e.g. e2e-testing.
Should we keep `experimental` as deprecated for now? @kpollich